docs/design/virtualization.md

```diff
@@ -41,14 +41,10 @@ Each hypervisor or VMM varies on how or if it handles each of these.
 
 ## Kata Containers Hypervisor and VMM support
 
-Kata Containers is designed to support multiple virtual machine monitors (VMMs) and hypervisors.
-Kata Containers supports:
- - [ACRN hypervisor](https://projectacrn.org/)
- - [Cloud Hypervisor](https://github.com/cloud-hypervisor/cloud-hypervisor)/[KVM](https://www.linux-kvm.org/page/Main_Page)
- - [Firecracker](https://github.com/firecracker-microvm/firecracker)/KVM
- - [QEMU](http://www.qemu-project.org/)/KVM
+Kata Containers
+[supports multiple virtual machine monitors (VMMs) and hypervisors](../hypervisors.md).
```

Comment:
I would remove "virtual machine monitors (VMMs) and". We tend to consistently call them hypervisors in Kata, see hypervisors.md.